annlite/core/index/hnsw/index.py

```diff
@@ -36,8 +36,14 @@ def pre_processed(self: 'HnswIndex', x: np.ndarray, *args, **kwargs):
             elif not self._set_backend_pq:
                 self._index.loadPQ(self.pq_codec)
                 self._set_backend_pq = True
+            dtables = self.pq_codec.get_dist_mat(x)
             x = self.pq_codec.encode(x)
-        return f(self, x, *args, **kwargs)
+
+            assert dtables.dtype == 'float32'
+            assert dtables.flags['C_CONTIGUOUS']
```

Comment:
Please remove the two asserts, I cannot see the benifits.

Reply:
Here assertions are to make sure the memory layout is expected. Since the `dtable`'s buffer will then be read as the float array and understood as the row-major storage style.
If those two asserts can't be satisfied, undefined behaviours may happen.